README.md

```diff
@@ -285,18 +285,18 @@ Including an example of how to use your role (for instance, with variables passe
          - { name: testu1 }
          - { name: testu2, state:present }
          - { name: testu3, state:absent }
-    clickhouse_clusters:
-      your_cluster_name:
-       shard_1:
-          - { host: "db_host_1", port: 9000 }
-          - { host: "db_host_2", port: 9000 }
-       shard_2:
-          - { host: "db_host_3", port: 9000 }
-          - { host: "db_host_4", port: 9000 }        
-    clickhouse_zookeeper_nodes:
-      - { host: "zoo_host_1", port: 2181 }
-      - { host: "zoo_host_2", port: 2181 }
-      - { host: "zoo_host_3", port: 2181 }
+      clickhouse_clusters:
+        your_cluster_name:
+        shard_1:
```

Comment:
The shard should be one level in as well

Reply:
Fixed